src/app/modules/cerf/cerf.component.ts

```diff
@@ -543,7 +544,11 @@ export class CerfComponent {
 
 	public getCerfFromForm() {
     let rawCerf = this.cerfForm.getRawValue();
+<<<<<<< HEAD
 		// Destructure the form in case
+=======
+		// Destructure the form in case 
+>>>>>>> a commentChange() function was added in order to enable the save button when adding anything to the comment fields
```

Comment:
looks like the merge markers screwed up here, will have to manually take these out